Drop SPARQLWrapper from the required install inputs under Python 2, as it
creates a circular dependency.

diff -u rdflib-4.1.2.alt/setup.py rdflib-4.1.2/setup.py
--- rdflib-4.1.2.alt/setup.py	2014-03-04 12:40:26.000000000 +0100
+++ rdflib-4.1.2/setup.py	2015-01-23 21:52:59.000000000 +0100
@@ -52,7 +52,7 @@
         kwargs['test_suite'] = "nose.collector"
         kwargs['install_requires'] = [
             'isodate',
-            'pyparsing', 'SPARQLWrapper']
+            'pyparsing']
 
         if sys.version_info[1]<7:  # Python 2.6
             kwargs['install_requires'].append('ordereddict')

